ztp/resource-generator/src/watcher.py

```diff
@@ -59,27 +59,25 @@ class PolicyGenWrapper(Logger):
     def __init__(self, paths: list):
         try:
             folders = [{'input': paths[0], 'output': paths[1]}]
-            src = '/usr/src/hook/cnf-features-deploy'
-            dest = '/tmp/cnf-features-deploy'
+            src = '/usr/src/hook/ztp'
+            dest = '/tmp/ztp'
+            shutil.rmtree(dest, ignore_errors=True)
             shutil.copytree(src, dest)
-            cwd = '/tmp/cnf-features-deploy/ztp/ztp-policy-generator'
-            command = 'kustomize build --enable-alpha-plugins'
-            oneliner_file = 'policyGenerator.yaml'
+            cwd = os.path.join(
+                '/tmp/ztp/ztp-policy-generator',
+                'kustomize/plugin/policyGenerator/v1/policygenerator/')
+            command = [
+                './PolicyGenerator',
+                'dummy_arg',
+                paths[0],
+                '../../../../../../source-cluster-crs',
```

Comment:
Since you know the path is rooted at /tmp/ztp it might be preferable to simply use /tmp/ztp/source-cluster-crs

Reply:
Yes, thanks for this note. Will do on my next PR. Don't want to mess up with the lgtm label :)